### PR TITLE
rTorrent: Add tracker scrape feature

### PR DESCRIPTION
**Pull request by stickz**
Resolves a software stability issue on rTorrent with tens of thousands of trackers. Announces trackers by default at regular intervals instead of forcefully announcing everything at startup.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ ___
 * [mktorrent](https://github.com/Rudde/mktorrent) installed for ruTorrent create plugin
 * [Traefik](https://github.com/containous/traefik-library-image) Docker image as reverse proxy and creation/renewal of Let's Encrypt certificates (see [this template](examples/traefik))
 * [geoip-updater](https://github.com/crazy-max/geoip-updater) Docker image to download MaxMind's GeoIP2 databases on a time-based schedule for geolocation
+* [rTorrent Patches](https://github.com/crazy-max/docker-rtorrent-rutorrent/tree/master/patches/rtorrent) Extended support for rTorrent to increase software stability.
```
Comment (crazy-max) [inline README.md:65]: We need an extra one for libtorrent right?

File: README.md [hunk 2]
```diff
@@ -342,6 +344,14 @@ Only torrent statistics are lost during a crash. (Ratio, Total Uploaded & Downlo
 Higher values will reduce disk usage, at the cost of minor stat loss during a crash.
 Consider increasing to 10800 seconds (3 hours) if running thousands of torrents.
 
+### rTorrent tracker scrape patch
```
Comment (crazy-max) [inline README.md:347]: Can you add it to TOC?

File: README.md (hunk 1 above)
Comment (stickz) [reply README.md:65]: libtorrent just has the 10gbps patch. I need to finalize that to make it work on the docker image. Will add in a sperate PR.

File: README.md (hunk 2 above)
Comment (stickz) [reply README.md:347]: What do you mean by TOC?

Comment (crazy-max) [reply README.md:347]: https://github.com/crazy-max/docker-rtorrent-rutorrent/blob/9e6dd21e94671e5137045a4b41e7e79227919d44/README.md?plain=1#L24-L47

Comment (stickz) [reply README.md:349]: Done.

**Pull request merged**